inc/thorvg.h

```diff
@@ -1048,7 +1048,7 @@ class TVG_EXPORT Picture final : public Paint
      *
      * @BETA_API
      */
-    const uint32_t* data() const noexcept;
+    const uint32_t* data(uint32_t* w, uint32_t* h) const noexcept;
```

Comment:
Same role as `size(uint32_t*, uint32_t*)`. Do we really need it?

Reply:
@JSUYA not same. size() is current(changed) size,  and this one is the original size.